sormas-app/app/src/main/java/de/symeda/sormas/app/event/edit/EventEditFragment.java

```diff
@@ -201,6 +203,13 @@ protected void prepareFragmentData() {
 		parenteralTransmissionModeList = DataUtils.getEnumItems(ParenteralTransmissionMode.class, true);
 		medicallyAssociatedTransmissionModeList = DataUtils.getEnumItems(MedicallyAssociatedTransmissionMode.class, true);
 		infectionPathCertaintyList = DataUtils.getEnumItems(InfectionPathCertainty.class, true);
+
+		List<SpecificRisk> specificRisks =
+				DatabaseHelper.getCustomizableEnumValueDao().getEnumValues(CustomizableEnumType.SPECIFIC_RISK);
+		specificRiskList = DataUtils.toItems(specificRisks);
+		if (record.getSpecificRisk() != null && !specificRisks.contains(record.getSpecificRisk())) {
+			specificRiskList.add(DataUtils.toItem(record.getSpecificRisk()));
+		}
```

Comment:
Make sure to not show the new field when there are no entries in the database; see CaseEditFragment.updateDiseaseVariantsField. You don't have to move the code to a dedicated method, just make sure that the visibility logic is executed (probably best to do that in onLayoutBinding)